changelog/v0.34.3/mem-client-expression-selector.yaml

```diff
@@ -0,0 +1,4 @@
+changelog:
+  - type: NON_USER_FACING
```

Comment:
It feels like since this is a library, this is user facing, in that we are now respecting ListOpts and WatchOpts on the in memory resource client?

Reply:
heh yeah, i saw that a lot of other 'user-facing' changes in solo-kit had NON_USER_FACING changelogs, so i went with that. but i can file an issue and link it here

Reply:
4129802